### PR TITLE
chore: modify readme grammar and spelling

### PR DESCRIPTION
**Pull request by madou**
Hi! Just did a once over for the readme to improve some things, let me know what you think.

Really loving the library! Need to give it a try soon.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -244,8 +244,8 @@ Since a `<Flex />` component works the same way as a DOM one, you can easily mak
 
 ### Measuring the container
 
-When building responsive layouts, you might need to syncronize the size of the 3D Flex container with the DOM, for example, to syncronize scroll position or the height of a scroll container.
-To make it easier, you can use the `onReflow` callback on the root `<Flex>` component, that will be called every time the flex layout is recalculated - e.g. when any content changes.
+When building responsive layouts you might need to syncronize the size of the 3D Flex container with the DOM, for example to synchronize scroll position or the modify the height of a scroll container.
```
Comment (giulioz) [inline README.md:247]: `or the modify the height`
is this right?

**Review by giulioz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (madou) [reply README.md:247]: yeah that was something I'm not sure about - both don't read easily

syncing the scroll position is obvious - what are we talking about when we mention height of a scroll container?

syncing the height of a scroll container to the scroll position..?

Comment (giulioz) [reply README.md:247]: I mean, isn't supposed to say _or **TO** modify_?

Comment (madou) [reply README.md:247]: hehe, yes that works

**Comment by giulioz** [comment]
Looks good to me!
Thanks for helping out, the library was written by italians, germans and russians after all 😝

**Pull request merged**